packages/map/src/map/utils.jsx

```diff
@@ -90,3 +90,11 @@ export const getFitBoundsOptions = ({
 
   return [boundingBox, options];
 };
+
+export const isWebGLSupported = (doc) => {
+  const canvas = doc.createElement('canvas');
+
+  // experimental-webgl for Edge browser
+  const gl = canvas.getContext('webgl') || canvas.getContext('experimental-webgl');
```

Comment:
did you research what is the best way to check for support? example: https://github.com/Modernizr/Modernizr/blob/acb3f0d990d3424ee7fade52796f9f97920354fc/feature-detects/webgl.js

also checking for support needs to be wrapped in try/catch, otherwise crashes:
https://github.com/Modernizr/Modernizr/blob/acb3f0d990d3424ee7fade52796f9f97920354fc/feature-detects/webgl/extensions.js

Reply:
`'WebGLRenderingContext' in window;`